src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -11,9 +11,9 @@
 
 namespace Symfony\Component\Serializer\Normalizer;
 
-use Symfony\Component\PropertyInfo\Type;
 use Symfony\Component\Serializer\Exception\InvalidArgumentException;
 use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+use Symfony\Component\TypeInfo\Type;
```

Comment:
This change means that if one loads serializer v7.1 with property-info v6.4, things will explode.
I'd rather make property-info required for serializer (same for validator)
Then we could move BackwardCompatibilityHelper to property-info, where it belongs IMHO (and not `@internal` - internal means "don't use cross-packages, and that applies cross-Symfony packages")